internal/config/config.go

```diff
@@ -57,6 +59,12 @@ func (c Configuration) Validate() error {
 			return err
 		}
 	}
+	if config, ok := interface{}(c.Preprocessing).(ConfigurationValidator); ok {
```

Comment:
Yeah, me neither, maybe this was part of a loop or something in another implementation and the check was needed because some configs didn't implement `ConfigurationValidator`.

Reply:
I changed this so it also returns all errors and not only the first one found.